server/config/config.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv').config()
+
+const MONGO_USERNAME = process.env.MONGO_USERNAME || ''
+const MONGO_PASSWORD = process.env.MONGO_PASSWORD || ''
+const MONGO_URI = `mongodb+srv://${MONGO_USERNAME}:${MONGO_PASSWORD}@cluster0.4bflh41.mongodb.net/BugStormDB?retryWrites=true&w=majority`
+const SERVER_PORT = process.env.PORT ? Number(process.env.PORT) : 5001
```

Comment:
i changed this to use port 5001, some mac has problems with port 5000 https://developer.apple.com/forums/thread/682332

Reply:
No problem :)